aci/resource_aci_vzsubj.go

```diff
@@ -25,6 +27,114 @@ func resourceAciContractSubject() *schema.Resource {
 
 		SchemaVersion: 1,
 
+		CustomizeDiff: func(_ context.Context, diff *schema.ResourceDiff, v interface{}) error {
+			result := false
+			if diff.HasChange("consumer_to_provider") {
+				old, new := diff.GetChange("consumer_to_provider")
+				oldInTerm := old.(*schema.Set).List()
+				newInTerm := new.(*schema.Set).List()
+				if len(newInTerm) == len(oldInTerm) {
+					oldInTermParam := oldInTerm[0].(map[string]interface{})
+					newInTermParam := newInTerm[0].(map[string]interface{})
+
+					if oldInTermParam["prio"] != newInTermParam["prio"] || oldInTermParam["target_dscp"] != newInTermParam["target_dscp"] || oldInTermParam["relation_vz_rs_in_term_graph_att"] != newInTermParam["relation_vz_rs_in_term_graph_att"] {
+						result = true
+					} else if oldInTermParam["relation_vz_rs_filt_att"] != newInTermParam["relation_vz_rs_filt_att"] {
```

Comment:
isn't this condition always true as directive is a Set and is represented by an ID not by the set itself?